packages/perseus/src/widgets/categorizer.jsx

```diff
@@ -285,6 +285,11 @@ const styles = StyleSheet.create({
         verticalAlign: "middle",
     },
 
+    emptyHeaderCell: {
+        backgroundColor: "white",
```

Comment:
question: Why is this necessary? Do we have a global style giving all `td`s a weird background color?

suggestion: Can you use something like `inherit` instead, in case someone tries to style a table and suddenly this cell stands out?

Reply:
Or `unset` or `currentColor` even maybe `transparent`... any number of others that don't hardcode a background color but default to whatever the test of the DOM is into